### DISCUSSION TITLE
PDF Meeting List not listing our Zoom Meetings

### DISCUSSION THREAD

**Discussion by PoconoIntergroup** [code4recovery/pdf#81] [Q&A]
They show on the Meeting Finder on our webpage and in the chair app, but we would like them to part of the PDF too.
Any help appreciated.

**Comment by tim-rohrer** [comment]
This is by design, but it is something the maintainer would be willing to discuss, I think. 

Long URLs don't work well with printed meeting lists, and our thinking is that most people who get printed lists are attending meetings in person (and perhaps getting phone numbers). 

I'm curious how you think people would use the information on a printed meeting list?

**Comment by PoconoIntergroup** [comment]
We have two different local Zoom meetings here. Both are listed in our printed meeting list with Zoom ID # and password instead of physical address. People can see it is zoom. Go to zoom app at the time and join the meeting with the Id and PW.It's not really much different. Location is just online. It works out for people who are sick or away but want to keep in touch with local people.Thanks for considering.On Sep 14, 2024 10:17 AM, Tim Rohrer ***@***.***> wrote:
This is by design, but it is something the maintainer would be willing to discuss, I think.
Long URLs don't work well with printed meeting lists, and our thinking is that most people who get printed lists are attending meetings in person (and perhaps getting phone numbers).
I'm curious how you think people would use the information on a printed meeting list?

**Reply by anchovie91471** [reply]
@PoconoIntergroup, we discussed this topic in our biweekly meeting and will be exploring some ways we can include online meetings in a user friendly way. I'll put updates on that effort here as we make progress.

Initially, we are going to include the "onl" (Online) type in the PDF, so users are made aware that "hybrid" meetings also have an online attendance option. They can then look on the service body's website to join. Many service bodies include a link to their website on the back or front cover of their booklet (or even a QR code) so anyone interested in joining online can visit the site and click to join, making it much easier for potential attendees.

I will update this discussion when that change is made.

Also, as a reminder, PDF is an open source project that can be downloaded, installed and customized to fit specific needs if you are so inclined. I know of a couple of folks that have done this.

Thanks for your feedback!
@anchovie91471
